src/napari_imagej/setup_imagej.py

```diff
@@ -0,0 +1,77 @@
+import os
+from jpype import JClass
+import logging
+import imagej
+from scyjava import config, jimport
+from threading import Thread, current_thread
+
+# -- LOGGER CONFIG -- #
+
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.DEBUG)  # TEMP
+
+# -- PUBLIC API -- #
+def imagej_init():
```

Comment:
Can we make this simpler?

`functools.lru_cache` is apparently [thread-safe](https://github.com/python/cpython/blob/8d999cbf4adea053be6dbb612b9844635c4dfb8e/Lib/functools.py#L470).

Maybe we can make `ij()` a `lru_cache` and put all of this initialization in there?

And then write
```
Thread(target=ij).start()
```
I don't think it would be that simple though.

This might be worth its own issue.